qt/python/mantidqt/mantidqt/widgets/sliceviewer/lineplots.py

```diff
@@ -312,6 +312,16 @@ def handle_key(self, key):
             self.exporter.export_pixel_cut(self._cursor_pos, key)
 
 
+class RectangleSelectorMtd(RectangleSelector):
+    def onmove(self, event):
+        """
+        Only process event if inside the axes with which the selector was init
+        This fixes bug #33264 where the x/y of the event originated form the line plot axes not the colorfill axes
```

Comment:
I don't think we should be having git issue numbers in doc strings. describing the issue should be sufficent.

Reply:
OK - any particular reason? I've seen this before and sometimes it's useful to understand the bug and get a MWE if there's a problem with that bit of code in the future.

Reply:
Will correct spelling mistake at the same time!